src/config/exocom_development_dependency.go

```diff
@@ -20,7 +20,8 @@ func (e *exocomDevelopmentDependency) compileServiceRoutes() ([]map[string]inter
 		return routes, err
 	}
 	serviceData := e.appConfig.GetServiceData()
-	for serviceName, serviceConfig := range serviceConfigs {
+	for _, serviceName := range e.appConfig.GetServiceNames() {
```

Comment:
Exocom's service routes were changing and causing false positives in the `main.tf` diffs.